### PR TITLE
Instruments in view

### PR DESCRIPTION
**Pull request by mennoo1996**
Builds on #185 
You can now see the selected instruments in the block and can change them in the detailview
Still needs styling

### CODE REVIEW

File: src/main/java/control/DetailViewController.java [hunk 1]
```diff
@@ -183,6 +188,71 @@ private void afterPaddingUpdateHelper() {
     }
     
     /**
+     * Initialize the instruments dropdown.
+     * @param shotBlock the shot block to do that for
+     */
+    private void initInstrumentsDropdown(ShotBlock shotBlock) {
+        log.error("Initing da dropdown");
```
Comment (geenen124) [inline src/main/java/control/DetailViewController.java]: Please be more descriptive with your logging...

File: src/main/java/control/DetailViewController.java [hunk 2]
```diff
@@ -383,42 +453,58 @@ void nameTextChangedListener(ObservableValue<? extends String> observable,
...
+     */
+    private void activeBlockChangedCamera() {
+        detailView = new DetailView();
+        detailView.setDescription(manager.getActiveShotBlock().getDescription());
+        detailView.setName(manager.getActiveShotBlock().getName());
+        detailView.setBeginCount(manager.getActiveShotBlock().getBeginCount());
+        detailView.setEndCount(manager.getActiveShotBlock().getEndCount());
+        initInstrumentsDropdown(manager.getActiveShotBlock());
+        detailView.setVisible();
+        detailView.setVisible(true);
```
Comment (geenen124) [inline src/main/java/control/DetailViewController.java]: Seems to be duplicate code (remove the one without the true?)

File: src/main/java/control/DetailViewController.java [hunk 3]
```diff
@@ -383,42 +453,58 @@ void nameTextChangedListener(ObservableValue<? extends String> observable,
...
+        detailView.setDescription(shotBlock.getDescription());
+        detailView.setName(shotBlock.getName());
+        detailView.setBeginCount(shotBlock.getBeginCount());
+        detailView.setEndCount(shotBlock.getEndCount());
+        ((DirectorDetailView) detailView).getPaddingBeforeField()
+            .setText(detailView.formatDouble(shotBlock.getPaddingBefore()));
+        ((DirectorDetailView) detailView).getPaddingAfterField()
+            .setText(detailView.formatDouble(shotBlock.getPaddingAfter()));
+        activeBlock = shotBlock;
+        //initDropDown(shotBlock);
```
Comment (geenen124) [inline src/main/java/control/DetailViewController.java]: Please remove commented code

File: src/main/java/control/DetailViewController.java (hunk 1 above)
Comment (Markvanderuit) [inline src/main/java/control/DetailViewController.java]: I don' see no problem, bro.

File: src/main/java/gui/centerarea/TimetableBlock.java [hunk 4]
```diff
@@ -117,6 +120,28 @@
...
+    }
+    
+    /**
+     * Remove an instrument from this timetable block.
+     * @param instrument the instrument to remove
+     */
+    public void removeInstrument(Instrument instrument) {
+        for (int i = 0;i < instrumentBox.getChildren().size(); i++) {
+            if (((Label) instrumentBox.getChildren().get(i)).getText()
+                    .equals(instrument.getName())) {
```
Comment (geenen124) [inline src/main/java/gui/centerarea/TimetableBlock.java]: Maybe extract this to a variable for readability/descriptiveness?

File: src/main/java/gui/headerarea/DetailView.java [hunk 5]
```diff
@@ -119,6 +129,19 @@ public void setBeginCount(double count) {
...
         endCountField.setText(formatDouble(count));
     }
+    
+    /**
+     * Set the instruments for this detail view.
+     * @param instruments the instruments to set
+     */
+    public void setInstruments(ArrayList<Instrument> instruments) {
+        System.out.println("Setting instruments");
+        System.out.println("size is " + instruments.size());
```
Comment (geenen124) [inline src/main/java/gui/headerarea/DetailView.java]: Please remove print statements or change them to log statements

File: src/main/java/gui/headerarea/DetailView.java [hunk 6]
```diff
@@ -131,6 +154,20 @@ private void initBeginCount() {
         beginCountBox.setAlignment(Pos.CENTER);
         itemBox.getChildren().add(beginCountBox);
     }
+    
+    /**
+     * Initialize the instrument dropdown.
+     */
+    private void initInstruments() {
+        System.out.println("Initing New ComboCheckBox");
```
Comment (geenen124) [inline src/main/java/gui/headerarea/DetailView.java]: Please remove print statements

**Comment by Markvanderuit** [comment]
Looks good. Please fix comments about dead code etc, then we'll merge.

**Pull request merged**